pkg/apis/configuration/validation/policy_test.go

```diff
@@ -1097,6 +1130,27 @@ func TestValidateURL(t *testing.T) {
 	}
 }
 
+func TestValidateQueryStringt(t *testing.T) {
+	t.Parallel()
+	validInput := []string{"foo=bar", "foo", "foo=bar&baz=zot", "foo=bar&foo=baz", "foo=bar%3Bbaz"}
```

Comment:
something like `foo=bar&baz=zot` is a valid input?

Reply:
The validation function is called after the query string is constructed, so that's the kind of string it would see if multiple args with different names were listed in the policy.

Reply:
ah right! It makes sense 🙂